distributed/utils_test.py

```diff
@@ -784,3 +795,54 @@ def captured_handler(handler):
         yield handler.stream
     finally:
         handler.stream = orig_stream
+
+
+@pytest.fixture(scope="function")
+def ssl_config(request):
+    """
+    In order to regenerate the cert files run the `continuous_integration/genereate_test_cert.sh` script
+
+    """
+    root = os.path.dirname(__file__)
+    certfile = os.path.join(root, 'tests', 'test.pem')
+    keyfile = os.path.join(root, 'tests', 'test.key')
+
+    assert os.path.exists(certfile)
+    assert os.path.exists(keyfile)
+
+    from .config import config
+
+    backup = {}
+    for k in ['default-scheme', 'tls-certfile', 'tls-keyfile']:
+        backup[k] = config[k]
```

Comment:
Should this be converted to `config.get(k, None)` to make these tests robust to this entry missing in the config file?  Generally we've so far not assumed that the config file has any values and instead inline defaults in code where appropriate.